src/App.jsx

```diff
@@ -25,21 +26,16 @@ export const alex = {
 
 export const App = () => (
   <div className="App">
-    <section className="Person">
-      <h2 className="Person__name">My name is Misha</h2>
-      <p className="Person__age">I am 37</p>
-      <p className="Person__partner">Natasha is my wife</p>
-    </section>
+    <Person
+      person={misha}
+    />
```

Comment:
```suggestion
    <Person person={misha} />
```
redundant to write prop on new line if it is just one prop and such short string.
fix also for other components